examples/rcbilling-demo/src/tests/main.test.ts

```diff
@@ -77,6 +79,38 @@ test.describe("Main", () => {
     const title = page.getByText("E2E Tests for Purchases JS");
     await expect(title).toBeVisible();
   });
+  test("Can render an RC Paywall using variables", async ({
```

Comment:
We probably should start splitting tests into multiple files 😅 but that can come in a separate PR.

Reply:
agreed (eventually)